Procfile

```diff
@@ -0,0 +1,2 @@
+release: python manage.py migrate
+web: gunicorn socialdist.wsgi --log-file -
```

Comment:
Procfile is under root folder, but manage.py is under backend folder, so I think there will be some problem...
Same as socialdist.wsgi